### DISCUSSION TITLE
whatsapp groups listing

### DISCUSSION THREAD

**Discussion by hamisirizwan** [WhiskeySockets/Baileys#503] [General Questions] [answered]
How do i fecth all whatsapp groups am in with baileys?, and get list of members

**Comment by Rippanteq7** [comment, marked as answer]
> How do i fecth all whatsapp groups am in with baileys?, and get list of members

To get all groups list, [use](https://github.com/WhiskeySockets/Baileys/blob/master/src/Socket/groups.ts#L33)
```js
sock.groupFetchAllParticipating()
```

Then you can use for loop to get list members

**Reply by hamisirizwan** [reply]
thanks and how can i identify a community and an individual group?
